src/main/java/LetterFrequencyReader/Main.java

```diff
@@ -0,0 +1,48 @@
+package LetterFrequencyReader;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Main {
```

Comment:
Pievērs uzmanību uzdevuma aprakstam un ņem vērā to, ka main klase ar main metodi faktiski prasīta netika. Būtu jāveido risinājums, kurš neietver main metodi, bet kurš tiek testēts tikai ar testu palīdzību. (Attiecīgi arī nosaukumu vajadzētu pamainīt no Main uz ko atbilstošaku)